subsys/bluetooth/controller/Kconfig.df

```diff
@@ -92,7 +92,7 @@ config BT_CTLR_DF_CTE_RX
 config BT_CTLR_DF_CONN_CTE_RSP
 	bool "Enable Connection CTE Response feature"
 	depends on BT_CTLR_DF_CTE_TX && BT_CONN
-	default y
+	default n
```

Comment:
Style nit: we don't do an explicit `default n` since this is the default.

```suggestion
```

Reply:
Thanks for spotting that, I've corrected it.